doc/user_guide/index.rst

```diff
@@ -293,11 +292,11 @@ A ``TLV.Message`` can be generated as follows:
 .. code:: ada
 
    with Ada.Text_IO;
-   with RFLX.RFLX_Builtin_Types; use type RFLX.RFLX_Builtin_Types.Length, RFLX.RFLX_Builtin_Types.Bytes;
+   with RFLX.RFLX_Types; use type RFLX.RFLX_Types.Bytes;
```

Comment:
Just a minor nitpick, do we want to keep `use` clauses as local as possible? In that case I'd move the `use type` into `Main`.

Reply:
Good idea. Changed.